src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -74,10 +74,16 @@ public function getProxyFactoryCode(Definition $definition, $id)
         $methodName = 'get'.Container::camelize($id).'Service';
         $proxyClass = $this->getProxyClassName($definition);
 
+        $generatedClass = $this->generateProxyClass($definition);
```

Comment:
this method internally now calls `getProxyClassName` as well (like two lines above). This should be avoided.